### ISSUE TITLE
Customize prompts?

### ISSUE THREAD

**Issue by sam1am** [brianpetro/obsidian-smart-connections#53]
Is there a way to customize the prompts sent to the API? It would be nice to be able to give it some specific direction or personality direction and a name and all that.

**Comment by brianpetro** [comment]
@sam1am yes! I'm trying to keep things simple for now, so we can find an ideal set of presets that works for average users. 

But enabling customizing prompts for advanced used is 100% possible.

Date/time inclusion is interesting. This may be able to be done automatically if time based references are detected in the user input.

Personality makes sense, too. Maybe this could be added without requiring editing the whole system prompt. Maybe append the default system prompt with some personality traits. 

Is there anything else prompt-wise that you had in mind? 

Thanks for your input and helping to make this plugin the best it can be!
